<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieWindow</class>
 <widget class="QMainWindow" name="MovieWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MovieWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Movie_Page">
       <attribute name="title">
        <string>Фильмы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="Search_Edit">
          <property name="placeholderText">
           <string>Введите название фильма</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Search_Button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Найти</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="Movie_output">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="Movie">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>907</width>
             <height>611</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Collections">
       <attribute name="title">
        <string>Подборки</string>
       </attribute>
       <widget class="QComboBox" name="CollectionsBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>201</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ButtonDeleteCollection">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ButtonAddMovie">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>10</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить фильмы</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ButtonAddCollection">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>10</y>
          <width>81</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NameCollectionEdit">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>10</y>
          <width>301</width>
          <height>25</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Введите название подборки</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="CollectionsMovieOutput">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>921</width>
          <height>641</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="CollectionMovie">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>919</width>
           <height>639</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Profile_Page">
       <attribute name="title">
        <string>Профиль</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="User_Label"/>
        </item>
        <item>
         <widget class="QLabel" name="Mail_Label"/>
        </item>
        <item>
         <widget class="QTableView" name="User_Review_Table"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
